docs/using/installing/index.rst

```diff
@@ -340,13 +348,22 @@ The file ``cluster.key`` should be kept only by the cluster administrator; it do
 
 You are now able to generate authentication certificates for the control service and each of your nodes.
 To generate the control service certificate, run the following command from the same directory containing your authority certificate generated in the previous step.
-Replace ``example.org`` with the hostname of your control service node; this hostname should match the one you will give to REST API clients.
+
+If you have a domain name for your cluster, replace ``example.org`` with the hostname of your control service node.
+You can also use an IP address here.
+For TLS verification purposes this hostname must match the one you will give to REST API clients.
 
 .. code-block:: console
 
    $ flocker-ca create-control-certificate example.org
    Created control-example.org.crt. Copy it over to /etc/flocker/control-service.crt on your control service machine and make sure to chmod 0600 it.
    
+Before copying these files you will need to SSH into the node and create a /etc/flocker directory:
+
+.. code-block:: console
+
+   root@mercury:~/$ mkdir -p /etc/flocker
```

Comment:
As I know you use chrome @davidoliverSP2 - test your prompt changes in Firefox to avoid confusion as there is a known bug. Prompt has a number of advantages, but mainly that the prompt is not copied even when selected, so you can drag over multiple lines and copy multiple commands.

In the context of these docs, `mercury` is `alice`'s local computer, to differentiate from `node`.